configs/config-test.json

```diff
@@ -92,6 +92,7 @@
       "auditingDatabaseName": "",
       "auditingDatabaseUserName": "",
       "auditingDatabasePassword": "",
+      "auditingDatabaseTableName": "",
```

Comment:
Can you add this to `configs/config-depl.json` and `configs/config-example.json` also?

Reply:
done